robot-server/Pipfile

```
@@ -7,32 +7,37 @@ name = "pypi"
 python_version = "3.7"
 
 [dev-packages]
-c445fee = {editable = true,path = "./../api"}
+opentrons-robot-server = {editable = true, path = "."}
```

We probably need to rename...

- `robot_server` to `opentrons_robot_server`
- `notify_server` to `opentrons_robot_server`
- `otupdate` to `opentrons_update_server`

...but for now, this should match its current setup.py name to avoid confusion:

```suggestion
robot-server = {editable = true, path = "."}
```